pyblish_lite/window.py

```diff
@@ -830,6 +831,17 @@ def on_was_processed(self, result):
             if instance.id not in models["instances"].ids:
                 models["instances"].append(instance)
 
+            family = instance.data['family']
+            if family:
+                plugins_filter = self.data['views']['right'].model()
+                plugins_filter.add_inclusion(role='families', value=family)
+
+            families = instance.data['families'] if 'families' in instance.data  else False
+            if families:
```

Comment:
Right, that's a better way, thanks.

The dictionary of inclusions is cleared during every reset, so every time the refresh button is clicked, with clear_inclusion() and :
```
    def reset(self):
        """Prepare GUI for reset"""
        self.info("About to reset..")

        plugins_filter = self.data["views"]["right"].model()
        plugins_filter.clear_inclusion()
        plugins_filter.add_inclusion(role="families", value="*")
```

So it clears the list of inclusions and then adds the "*". Maybe the "*" should be added during clear_inclusion() command